fab/fabfile.py

```diff
@@ -551,6 +519,14 @@ def record_successful_deploy(url):
 
 
 @task
+@roles(ROLES_ALL_SRC)
+@parallel
+def record_successful_release():
+    with cd(env.root):
+        files.append(RELEASE_RECORD, str(env.code_root), use_sudo=True)
```

Comment:
Can we add some data to this like the user who deployed, the git hash, maybe even how long it took.